server/src/main/java/com/cloud/deploy/DeploymentPlanningManagerImpl.java

```diff
@@ -406,7 +408,10 @@ public DeployDestination planDeployment(VirtualMachineProfile vmProfile, Deploym
             planner = getDeploymentPlannerByName(plannerName);
         }
 
-        if (vm.getLastHostId() != null && haVmTag == null) {
+        String considerLastHostStr = (String)vmProfile.getParameter(VirtualMachineProfile.Param.ConsiderLastHost);
+        boolean considerLastHost = vm.getLastHostId() != null && haVmTag == null &&
+                (considerLastHostStr == null || Boolean.TRUE.toString().equalsIgnoreCase(considerLastHostStr));
```

Comment:
```suggestion
        boolean considerLastHost = vm.getLastHostId() != null && haVmTag == null &&
                Boolean.TRUE.toString().equalsIgnoreCase(considerLastHostStr);
```
I think we should maintain the current default behavior. That is, if this parameter is not informed, the considerLastHost should be false.

Reply:
What's the thought behind it? Any downsides on considering last host?

Reply:
@JoaoJandre 
see above. the current behavior is same as considerLastHost=true

Reply:
Same as above.